CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Fixed `post-robot` package version at `10.0.35`.
```

Comment:
What do you think of an entry in the readme to let future maintainers know about this fixed version?

Reply:
Hm... I don't think the readme is the best place for an implementation detail like this. Do you have other suggestion as to where to add this information? I was hoping future maintainers would trace back to this PR to find out the reason this version is fixed...

Reply:
Maybe a new .md file like development.md or similar? But if you think that the trace is enough good with me.

Reply:
Approved anyway since this talk is diverging from the main purpose.

Reply:
I explained the reason in the changelog.